keypairs/secp256k1.go

```diff
@@ -0,0 +1,95 @@
+package keypairs
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcec/v2/ecdsa"
+)
+
+const SECP256K1_PREFIX = "00"
+
+type secp256k1Alg struct{}
+
+func (c *secp256k1Alg) deriveKeypair(decodedSeed []byte, validator bool) (string, string, error) {
```

Comment:
Hi @GuillemGarciaDev , I am working on the secp256k1 keypairs and I still get incorrect results for the `deriveKeypair` function (forget the sign and validate for now).
I wanted to check with you if you see something wrong in what I am doing ? I am trying to copy what has been done in xrpl.js. Thank you